catalog-info.yaml

```diff
@@ -363,4 +363,4 @@ spec:
         release-eng:
           access_level: BUILD_AND_READ
         everyone:
-          access_level: READ
+          access_level: READ_ONLY
```

Comment:
I think I took the above access level called  `BUILD_AND_READ`  and removed `BUILD_AND`, assuming it was enough :/ .  It's undoubtedly misleading that `_ONLY` is needed; what else will be read if it's only read? Thanks for the fix Linh!